ddt.py

```diff
@@ -110,7 +137,7 @@ def wrapper(func):
     return wrapper
 
 
-def mk_test_name(name, value, index=0):
+def mk_test_name(name, value, index=0, fmt_test_name=FormatTestName.DEFAULT):
```

Comment:
We could limi it with `name_fmt` or even `fmt` as the function already is about test naming.

Reply:
Updated and you will see the changes in the next commit.  (I picked `name_fmt`)